cmd/agent/main.go

```diff
@@ -2,21 +2,28 @@ package main
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"time"
 
 	"github.com/mihailtudos/metrickit/internal/config"
 	"github.com/mihailtudos/metrickit/internal/domain/repositories"
 	"github.com/mihailtudos/metrickit/internal/infrastructure/storage"
 	"github.com/mihailtudos/metrickit/internal/service/agent"
+	"github.com/mihailtudos/metrickit/pkg/helpers"
 )
 
 func main() {
 	agentCfg, err := config.NewAgentConfig()
 	if err != nil {
-		agentCfg.Log.ErrorContext(context.Background(), "failed to get agent configurations: "+err.Error())
+		fmt.Println(err)
```

Comment:
You should not use `fmt.Println` in your code. If you need to log something either use your logger or, if it is not yet initialized, the `log` package